db/migrate/20240218181609_change_project_media_columns_for_clusters.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeProjectMediaColumnsForClusters < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :clusters, :project_medias_count
+    add_reference :clusters, :project_media, index: true # Center
```

Comment:
No need to add this column again as we should remove the line that deleted this column for  `20240212055200_create_cluster_project_medias.rb` migration as column already exists so need need to delete the column and recreate it again